doc/api/dgram.md

```diff
@@ -239,8 +239,9 @@ respect to [byte length][] and not the character position.
 If `msg` is an array, `offset` and `length` must not be specified.
 
 The `address` argument is a string. If the value of `address` is a host name,
-DNS will be used to resolve the address of the host. If the `address` is not
-specified or is an empty string, `'127.0.0.1'` or `'::1'` will be used instead.
+DNS will be used to resolve the address of the host. Otherwise, if `address` is
+not specified or is an empty string, `'127.0.0.1'` (`udp4`) or `'::1'` (`udp6`)
```

Comment:
> @boneskull I don't see the usefulness in permitting values like false and 0 and treating them as undefined, if that is what you are suggesting?

This is the current behavior.  Values are passed through to `dns.lookup()`, which allows such things.  The documentation for `dgram` is therefore inaccurate.  And ambiguous, which prompted the change.

> I think he is saying that is current behaviour, and not doing it would be semver-major. I would say if we are going to change the API, I would expect only allow null and undefined to mean "no value"

This is correct.  I'm not proposing making a semver-major change here.  Whether or not allowing `false` is "useful", I don't know, but I'd argue that's not a big enough deal to warrant a semver-major change.

> Btw, from above, it looks like 42 would also be considered a host name? I.e., numbers are allowed? Do they get converted to strings, like '42' and passed to the resolver?

No.  The only valid `number` is `0`, which is falsy.  Then:

> If it's any other truthy value (except Function; see below), it's invalid, and send() should throw a TypeError.